### PR TITLE
fingerprint input fixed

### PR DESCRIPTION
**Pull request by LeSiiN**
- added placeholder
- added maxlength ( length of fingerprint id )
- fixed ,,none'' fingerprint when saving
- removed print

### CODE REVIEW

File: server/main.lua [hunk 1]
```diff
@@ -346,7 +346,7 @@ QBCore.Functions.CreateCallback('mdt:server:GetProfileData', function(source, cb
 		pp = ProfPic(target.charinfo.gender),
 		licences = licencesdata,
 		dob = target.charinfo.birthdate,
-		fingerprint = target.metadata.fingerprint,
+		fingerprint = '',
```
Comment (xFutte) [inline server/main.lua:349]: Couldn't this just be
```suggestion
		fingerprint = target.metadata.fingerprint or '',

```
So the '' is a fallback if the first value returns nil?

Comment (LeSiiN) [reply server/main.lua:349]: no because then it gets added automatically again, its the same for the mdtinfo

Comment (xFutte) [reply server/main.lua:349]: Oh, I put it in the wrong context. Woops.

**Comment by MonkeyWhisper** [comment]
> * added placeholder
> * added maxlength ( length of fingerprint id )
> * fixed ,,none'' fingerprint when saving
> * removed print

What is wrong with the current fingerprint? I didn't see any issue when I was testing it, I do see a place holder

**Comment by LeSiiN** [comment]
> > * added placeholder
> > * added maxlength ( length of fingerprint id )
> > * fixed ,,none'' fingerprint when saving
> > * removed print
> 
> What is wrong with the current fingerprint? I didn't see any issue when I was testing it, I do see a place holder

So for me it was not saving the fingerprint correctly because it took the metadata instantly instead of the one from database

**Comment by MonkeyWhisper** [comment]
> > > * added placeholder
> > > * added maxlength ( length of fingerprint id )
> > > * fixed ,,none'' fingerprint when saving
> > > * removed print
> > 
> > 
> > What is wrong with the current fingerprint? I didn't see any issue when I was testing it, I do see a place holder
> 
> So for me it was not saving the fingerprint correctly because it took the metadata instantly instead of the one from database

This wasn't happening to me, did you update? I don't think this is needed, it works pretty well and no one has reported that. 

Also, I checked this PR and there are two issues

1 - Before even searching for a profile, the empty fingerprint appears
image

2 - The search function is grabbing the first profile fingerprint before selecting profile

image

**Comment by MonkeyWhisper** [comment]
The second issue is happening with the current latest version, I'll send a PR to fix that

**Comment by MonkeyWhisper** [comment]
Fixed second already with PR

**Pull request closed without merging**